apps/organizations/models.py

```diff
@@ -209,6 +209,13 @@ class Meta:
             " are always sent via NRE regardless of this setting"
         ),
     )
+    disable_reminder_emails = models.BooleanField(
+        default=False,
+        help_text=(
+            "If True, annual contribution receipt email assumed is not sent through RevEngine. Other emails, e.g. magic_link,"
+            " are always sent via NRE regardless of this setting"
+        ),
```

Comment:
Can this be rephrased a little?

```suggestion
            "If True, annual contribution reminder emails will not be sent through RevEngine. "
            "This does not impact other transactional emails."
        ),
```